cmd/di.go

```diff
@@ -329,10 +331,22 @@ func (di *Dependencies) bootstrapNetworkComponents(options node.OptionsNetwork)
 		network.EtherClientRPC = options.EtherClientRPC
 	}
 
+	if options.StatisticsAddress != metadata.DefaultNetwork.ELKAddress {
```

Comment:
I don't like `ELK` name here since it only ElasticSearch and there is no Logstash and Kibana.

Reply:
Renamed name to `ElasticSearch`, and remove the name altogether where appropriate. But it makes sense to name transport with the name (`ElasticSearchTransport`), since the implementation is specific to elastic search.